occrp/templates/partials/facets.html

```diff
@@ -0,0 +1,24 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h4 class="panel-title">
+      <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#collapse{{facet}}">
+        {{facet}}
+      </a>
+      <span class="pull-right"><i class="fa fa-plus" aria-hidden="true"></i></span>
```

Comment:
We should probably bind a handler to the `show.bs.collapse` and `hide.bs.collapse` to replace the ➕  with a ➖ here. Also, I'd put this span inside the anchor tag so that clicking anywhere on the panel will show or hide the content (not just on the words).